samples/app-complete-auth/csharp/M365Agent/m365agents.local.yml

```diff
@@ -27,15 +28,13 @@ provision:
       name: app-complete-auth-${{TEAMSFX_ENV}}
     # Write the information of created resources into environment file for
     # the specified environment variable(s).
-    writeToEnvironmentFile: 
+    writeToEnvironmentFile:
       teamsAppId: TEAMS_APP_ID
 
   - uses: script
     with:
-      run:
-        # echo "::set-teamsfx-env MICROSOFT_APP_TYPE=SingleTenant";
-        echo "::set-teamsfx-env MICROSOFT_APP_TYPE=MultiTenant";
-        echo "::set-teamsfx-env MICROSOFT_APP_TENANT_ID=${{AAD_APP_TENANT_ID}}";
+      run: echo "::set-teamsfx-env MICROSOFT_APP_TYPE=SingleTenant"; echo
+        "::set-teamsfx-env MICROSOFT_APP_TENANT_ID=${{AAD_APP_TENANT_ID}}";
```

Comment:
Improper order please make it correct like this

**run:**
<img width="376" height="42" alt="image" src="https://github.com/user-attachments/assets/323126a7-c2a8-4056-8266-4662ab052762" />